core/src/com/dragons/game/model/ISubject.java

```diff
@@ -1,8 +1,27 @@
 package com.dragons.game.model;
 
+import com.badlogic.gdx.math.Circle;
+import com.badlogic.gdx.math.Shape2D;
+import com.badlogic.gdx.math.Vector;
+import com.badlogic.gdx.math.Vector2;
 import com.dragons.game.view.modelViews.IModelObserver;
 
+import java.awt.Shape;
+
+/**
+ * The ISubject interface is used to implement our top-level observer type that will be common
+ * for all game-classes that may implement the observer pattern.
+ *
+ * @author Jakob Eikeland
+ * */
+
 public interface ISubject {
```

Comment:
Alle modeller implementerer denne interfacen. Se på det som en kontrakt som sier at alle "ting" må ha minst disse metodene! Posisjon, en form (enten rektangel eller sirkel) + mulighet for å legge til observatorer.